src/database/database.ts

```diff
@@ -0,0 +1,21 @@
+import { Sequelize } from 'sequelize';
+import dotenv from 'dotenv';
+dotenv.config();
+
+const sequelize = new Sequelize(`${process.env.DATABASE_URL}`, {
```

Comment:
Add necessary environment variables in `.env.example` file, e.g., NODE_ENV, DATABASE_URL, TEST_DATABASE_URL, PROD_DATABASE_URL and provide one sentence explaining each